.github/workflows/e2e-tests-workflow.yml

```diff
@@ -66,14 +68,14 @@ jobs:
           plugin_version=$(node -pe "require('./package.json').version")
           echo plugin version: $plugin_version
           echo plugin name: $AD_OPENSEARCH_DASHBOARDS_PLUGIN_NAME
-          if docker pull $OPENSEARCH_DOCKER_IMAGE:latest
+          if docker pull $OPENSEARCH_DOCKER_IMAGE:$OPENSEARCH_DOCKER_TAG
           then
             ## Populate the Dockerfiles
-            echo "FROM $OPENSEARCH_DOCKER_IMAGE:latest" >> Dockerfile-AD-OpenSearch
-            echo "FROM $DASHBOARDS_DOCKER_IMAGE:latest" >> Dockerfile-AD-OpenSearch-Dashboards
+            echo "FROM $OPENSEARCH_DOCKER_IMAGE:$OPENSEARCH_DOCKER_TAG" >> Dockerfile-AD-OpenSearch
+            echo "FROM $DASHBOARDS_DOCKER_IMAGE:$DASHBOARDS_DOCKER_TAG" >> Dockerfile-AD-OpenSearch-Dashboards
             echo "COPY build/$AD_OPENSEARCH_DASHBOARDS_PLUGIN_NAME-$plugin_version.zip ." >> Dockerfile-AD-OpenSearch-Dashboards
             ## Uninstall existing AD artifact and install new one
-            echo "RUN if [ -d /usr/share/opensearch-dashboards/plugins/$AD_OPENSEARCH_DASHBOARDS_PLUGIN_NAME ]; then /usr/share/opensearch-dashboards/bin/opensearch-dashboards-plugin remove $AD_OPENSEARCH_DASHBOARDS_PLUGIN_NAME; fi" >> Dockerfile-AD-OpenSearch-Dashboards
+            echo "RUN if [ -d /usr/share/opensearch-dashboards/plugins/anomaly-detection-dashboards ]; then /usr/share/opensearch-dashboards/bin/opensearch-dashboards-plugin remove anomaly-detection-dashboards; fi" >> Dockerfile-AD-OpenSearch-Dashboards
```

Comment:
This is due to the default plugin names being installed as kebab-case instead of camelCase in Dashboards ([PR](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/357)). Eventually, we will change the `AD_OPENSEARCH_DASHBOARDS_PLUGIN_NAME`, which is the zip name, to be consistent and kebab-case as well.